lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/LibertyXSDURIResolver.java

```diff
@@ -111,37 +115,45 @@ private String generateServerSchemaXsd(LibertyWorkspace libertyWorkspace, Path s
     //java -jar path/to/ws-schemagen.jar path/to/workspace/.libertyls/libertySchema.xsd
     File tempDir = LibertyUtils.getTempDir(libertyWorkspace.getWorkspaceURI().toString());
 
+    //If tempDir is null, generate the schema file in the same directory as ws-schemagen.jar 
+    if (tempDir == null) {
+      tempDir = new File(schemaGenJarPath.toString()).getParentFile();
```

Comment:
Wouldn't this generate it in the Liberty installation itself? That doesn't seem like a good idea.

Reply:
Yeah, it would. What would be a better alternative for a default? The `server.xml` location?